assets/src/js/annotation/backend.js

```diff
@@ -0,0 +1,25 @@
+const API_HOST = process.env.TIER === 'production'
```

Comment:
@ShishKabab I don't know where should I put this for checking the development and production version. :slightly_smiling_face:

Reply:
I'd recommend creating a function in the root utils.js getDeploymentTier() that checks whether the current window.location.hostname is localhost (development) or starts with staging. or returns production otherwise.